src/Result/inc/Result.h

```diff
@@ -0,0 +1,24 @@
+////////////////////////////////////////////////////////////////////////////////
+/**
+* @file Result.h
+* @brief Result Object function declarations.
+*/
+////////////////////////////////////////////////////////////////////////////////
+#include "Parser.h"
+#include <string>
+
+#ifndef RESULT_H
+#define RESULT_H
+
+namespace calculator
+{
+    class Result
+    {
+        public:
+            Expression expression;
+            float answer;
+            std::string fullResult;
+            void returnResult();
```

Comment:
Needs Doxygen comment

Reply:
function is called returnResult() but doesn't return anything

consider renaming to getFullResult() and then having main() or elsewhere output that to cout